test/unit/template_importer_test.rb

```diff
@@ -225,18 +225,57 @@ def self.import!(name, text, _metadata)
       end
     end
 
+    test 'should import files from git' do
+      setup_settings
+      imp = ForemanTemplates::TemplateImporter.new
```

Comment:
Ah cool, that'll simplify tests in the future too, since it can use `import_from_files`. I'll give it a quick test but it's looking good!